apps/myaccount/src/pages/account-security.tsx

```diff
@@ -143,6 +144,21 @@ const AccountSecurityPage = (props: RouteProps): ReactElement => {
                         </Grid.Row>
                     ) : null }
 
+                { hasRequiredScopes(accessConfig?.security, accessConfig?.security?.scopes?.read, allowedScopes) &&
+                    isFeatureEnabled(
```

Comment:
```suggestion
                    isFeatureEnabled (
```

Reply:
here i followed the same format used in other components in the account-security.tsx